src/bp/core/services/hook/hook-service.ts

```diff
@@ -99,6 +100,18 @@ export namespace Hooks {
       this.folder = 'before_suggestions_election'
     }
   }
+
+  export class OnIncidentStatusChanged implements BaseHook {
+    folder: string
```

Comment:
what's that "folder" property ?

Reply:
Every hook has a folder property, to set where the hook files should be located